Add blog post for version 2.35.0 (#94)
scm-manager/website: content/posts/2022-06-01-scm-manager-2-35-0/index.md

author: René Pfeuffer
categories:
- scm-manager
- release
keywords:
- plugins
- java
- startup
---

Hi SCM-Manager Community,

with the new release of SCM-Manager we made some significant changes we have to tell you about. Most importantly, if
you still use Java 8, now is the time to update your system.

## Java 11

From version 2.35.0 on, SCM-Manager has to be run at least with Java 11. If you still use Java 8, this is the time to
update, not only due to SCM-Manager, but also because Java 8 is out of maintenance. We have anticipated this for a long
time, because now we can use some nice new features of the more current Java releases.

## Plugin Wizard

For those of you that are about to set up a new server, you will notice an additional initialization page: a plugin
wizard. We have thought about different use cases of SCM-Manager from personal usage up to big development teams.
We've done this, so that you can get the best experience possible from your SCM-Manager.

![The new plugin wizard page](assets/plugin-wizard.png)

## Command Message Check

For some time now, we have the [Commit Message Checker](https://scm-manager.org/plugins/scm-commit-message-checker-plugin/).

> With this, you can make sure that your commit messages apply to some rules. Though this is useful, it still may be

Reviewer: ```suggestion
With this, you can make sure that your commit messages comply with pre-defined rules. Though this is useful, it still may be
```

tedious, when you have created a bunch of commits locally before pushing them and just then being reminded about the

> rules. With our new [CLI client](https://scm-manager.org/cli/) it is now possible to create a pre commit hook that will

Reviewer: ```suggestion
rules. With our new [CLI client](https://scm-manager.org/cli/) it is now possible to create a pre-commit hook that will
```
